components/compositing/compositor.rs

```diff
@@ -756,18 +614,7 @@ impl<Window: WindowMethods> IOCompositor<Window> {
             }
 
             (Msg::CollectMemoryReports(reports_chan), ShutdownState::NotShuttingDown) => {
```

Comment:
At this point we could also remove this message entirely. Please do it or file a followup for it.